code/game/objects/items/shooting_range.dm

```diff
@@ -79,102 +77,3 @@
 	icon_state = "target_q"
 	desc = "A shooting target that looks like a xenomorphic alien."
 	hp = 2350 // alium onest too kinda
-
-/obj/item/target/bullet_act(var/obj/item/projectile/Proj)

```

Comment:
Improve, don't remove. We have mutable appearances for a reason, these days.

Reply:
no clue how to use them, game stability is more important than tiny bullet holes on a target

Reply:
/TG/ fixed whatever was causing their canvases to crash players. This is related to it (at least I think it is) so shouldn't we be looking at what they did?

Reply:
@taukausanake you're free to put up a PR